### PR TITLE
Proxying and Cookie bug fix

### PR DESCRIPTION
**Pull request by UladBohdan**
* Custom server using Next.
* Proxying API queries through Frontend.
* Cookie bug fixed.

### CODE REVIEW

**Comment by UladBohdan** [comment]
@Drapegnik review the PR please.

File: package.json [hunk 1]
```diff
@@ -4,11 +4,10 @@
   "description": "Next.js & React app",
   "scripts": {
     "start": "npm run dev",
-    "dev": "next -p 3000",
+    "dev": "node server.js",
```
Comment (Drapegnik) [inline package.json]: Are we can specify port here for explicity?

File: utils/request.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ export const DEFAULT_OPTIONS = {
     Accept: 'application/json',
     'Content-Type': 'application/json',
   },
+  credentials: 'same-origin',
```
Comment (Drapegnik) [inline utils/request.js:7]: 👽

**Comment by Drapegnik** [comment]
TODO: add fetching current user on init

File: package.json (hunk 1 above)
Comment (UladBohdan) [reply package.json]: we can add something like that:
`export PORT=3000 && node server.js`
makes sense: that ensures `PORT` variable set e.g. for Backend won't affect Frontend.

**Comment by Drapegnik** [comment]
[auth cookies bugs](https://trello.com/c/HGRitZDA/76-auth-cookies-bugs)

**Comment by Drapegnik** [comment]
[to identify user on login if session cookie is presented](https://trello.com/c/5OwMkyo6/84-to-identify-user-on-login-if-session-cookie-is-presented)

**Comment by UladBohdan** [comment]
1. is that fine the `login` button on main page remains present for about a second and only then is replaced with "Hello, username" message ?
2. could you please add `log out` button on main page? I've checked it works fine by manually calling `/auth/logout` although having a button if we're logged in would be nice

**Comment by Drapegnik** [comment]
@UladBohdan:
1. it takes some time for request on server, it's ok
2. done!

**Comment by UladBohdan** [comment]
Looks and works good, thanks! Let's merge it.

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**